### PR TITLE
WDS-74 - ProgressBar component

### PR DESCRIPTION
**Pull request by timofeytrepalin**

### CODE REVIEW

File: packages/ui/src/components/ProgressBar/SProgressBar.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    lineHeight?: number
+  }>(),
+  {
+    percent: 0,
+    lineHeight: 2
+  }
+)
+
+  const activeBarWidth = computed(()=> {
+    return props.percent < 0
```
Comment (0x009922) [inline packages/ui/src/components/ProgressBar/SProgressBar.vue:15]: Not critical, but I would rewrite it like this:

```js
`${Math.min(Math.max(props.percent, 0), 100)}%`
```

or

```js
const val = props.percent;
`${val < 0 ? 0 : val > 100 ? 100 : val}%`
```

**Review by 0x009922 — APPROVED** [review]

File: packages/ui/src/components/ProgressBar/SProgressBar.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.s-progress-bar {
+  height: $progress-bar-height;
+  width: 100%;
+  background: var(--s-color-base-border-secondary);
+  border-radius: $progress-bar-border-radius;
+
+  &__active-bar {
+    height: 100%;
+    border-radius: $progress-bar-border-radius;
+    background: linear-gradient(90deg, var(--s-color-theme-accent) $active-bar-width, transparent 0%);
```
Comment (0x009922) [inline packages/ui/src/components/ProgressBar/SProgressBar.vue:53]: Please use correct theme binding

**Review by 0x009922 — CHANGES_REQUESTED** [review]

**Comment by 0x009922** [comment]
Also please add a minor changeset

File: .changeset/config.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "commit": false,
   "linked": [],
   "access": "restricted",
-  "baseBranch": "master",
```
Comment (0x009922) [inline .changeset/config.json:7]: Don't include this change, please

**Review by 0x009922 — APPROVED** [review]

**Review by stefashkaa — APPROVED** [review]

**Pull request merged**